recipe/meta.yaml

```
@@ -1,22 +1,21 @@
 {% set name = "PyTrilinos" %}
-{% set version = "12.10.1" %}
-{% set sha256 = "ea8ff573bb1f452f2585950cdabe7e2c84cb35013844c58dc878c66bd4245e06" %}
+{% set version = "12.12.1" %}
+{% set dashversion = "12-12-1" %}
+{% set sha256 = "5474c5329c6309224a7e1726cf6f0d855025b2042959e4e2be2748bd6bb49e18" %}
 {% set mpi = 'mpich 3.2.*' %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  fn: trilinos-{{ version }}-Source.tar.gz
-  url: http://trilinos.csbsju.edu/download/files/trilinos-{{ version }}-Source.tar.gz
+  fn: trilinos-release-{{ dashversion }}.tar.gz
+  url: http://github.com/trilinos/Trilinos/archive/trilinos-release-{{ dashversion }}.tar.gz
   sha256: {{ sha256 }}
-  patches:
-    - 12.10-fixes.patch
 
 build:
-  number: 2
-  skip: true  # [win or py3k]
+  number: 0
+  skip: True  # [ osx or win or py3k ]
```

```suggestion
  skip: True  # [ osx or win]
```

It seems 12.12 is now compatible with python 3